jax/_src/numpy/lax_numpy.py

```diff
@@ -1197,13 +1197,13 @@ def _split(op: str, ary: ArrayLike, indices_or_sections: Union[int, ArrayLike],
                                                  f"in jax.numpy.{op} argument 1")
     part_size, r = _divmod(size, indices_or_sections)  # type: ignore[misc]
     if r == 0:
-      split_indices = np.arange(indices_or_sections + 1,
-                                dtype=np.int64) * part_size
+      split_indices = [np.int64(i) * part_size  # type: ignore
```

Comment:
Did you have to change this because it uses numpy, which no longer supported for symbolic dimensions?